2048/2048/src/main/java/com/appcoins/eskills2048/activity/FinishGameActivity.java

```diff
@@ -91,7 +121,20 @@ private void showLoading() {
     binding.retryButton.setVisibility(View.GONE);
   }
 
+  private void buildRecyclerView(RoomResponse roomResponse) {
```

Comment:
you can run this method when creating the activity and update the recyclerview as soon as you have the data